crates/hyperdrive-math/src/short/max.rs

```diff
@@ -519,14 +548,17 @@ mod tests {
     /// functions are equivalent.
     #[tokio::test]
     async fn fuzz_calculate_max_short_no_budget() -> Result<()> {
```

Comment:
This name could be improved IMO. It's not obvious that this is a differential test and the other isn't a differential test.

Reply:
A related concern that I have with this test is that it's not verifying the invariants that we verify in the other test. I like the change that you made to the other test, but there should still be a test that verifies that our invariants hold with cases that aren't budget constrained.

Reply:
> This name could be improved IMO. It's not obvious that this is a differential test and the other isn't a differential test.

agreed. I've renamed a bunch of tests to try to have consistency with `fuzz` == "runs executions with random inputs in a loop over `FUZZ_RUNS` or `FAST_FUZZ_RUNS`". vs `test` == "An isolated test run once". I think it makes sense to add a third name, `diff` or `comp` or `sol`, perhaps, that we can add to `fuzz`.

If you have a preference lets go with it, I don't. If you don't then I'll pick one and follow up with a mass rename in another PR.

Reply:
> A related concern that I have with this test is that it's not verifying the invariants that we verify in the other test. I like the change that you made to the other test, but there should still be a test that verifies that our invariants hold with cases that aren't budget constrained.

Sure, what invariant checks should I add?

Reply:
I like `diff` or `sol`, but it's totally up to you

Reply:
went with sol